packages/synapse-react-client/src/components/SynapseTable/SearchV2.tsx

```diff
@@ -164,14 +164,25 @@ class _Search extends React.Component<InternalSearchProps, SearchState> {
           }
         additionalFilters.push(columnMultiValueQueryFilter)
       } else {
-        const columnSingleValueQueryFilter: ColumnSingleValueQueryFilter = {
-          columnName,
-          operator: ColumnSingleValueFilterOperator.LIKE,
-          values: [`%${searchText}%`],
-          concreteType:
-            'org.sagebionetworks.repo.model.table.ColumnSingleValueQueryFilter',
+        if (columnModel?.columnType == ColumnTypeEnum.ENTITYID) {
```

Comment:
For an entity ID type column, add an additionalFilter where we strip the 'syn' from the search text